### PR TITLE
Adding edits

### PR DESCRIPTION
**Pull request by dbialek2**
Requested by @pdarvasi in https://github.com/sequenceiq/cloudbreak-docs/pull/177

@pdarvasi Please review the suggested edits. Sorry for being late with this.

### CODE REVIEW

File: docs/azure/ui/credential.md [hunk 1]
```diff
@@ -1,93 +1,87 @@
...
 
-1. Fill out the new credential `Name`
-    * Only alphanumeric and lowercase characters (min 5, max 100 characters) can be applied
+1. Provide the `Name` for your new credential
+    * Only alphanumeric and lowercase characters (min 5, max 100 characters) can be used
   
 2. Copy your SSH public key to the `SSH public key` field
-    * The SSH public key must be in OpenSSH format and it's private keypair can be used later to [SSH onto every instance](operations.md#ssh-to-the-hosts) of every cluster you'll create with this credential.
-      - The **SSH username** for the AZURE instances is **cloudbreak**.
+    * The SSH public key must be in the OpenSSH format. The matching private key can be used later to [SSH onto every instance](operations.md#ssh-to-the-hosts) of every cluster that you'll create with this credential. The **SSH username** for the AZzure instances is always **cloudbreak**.
```
Comment (pdarvasi) [inline docs/azure/ui/credential.md]: AZzure -> Azure typo

File: docs/azure/ui/credential.md [hunk 2]
```diff
@@ -1,93 +1,87 @@
...
         * This is the default behaviour which needs no further input and will assign Cloudbreak service principal `Contributor` role for your subscription
     * `Reuse existing custom role`
-        * You can reuse an already existing Azure role which has the required minimal permission set necessary for Cloudbreak to be able the manage the cluster's resources. It returns an error if no role with the name specified exists or the role does not have the required permission set. 
+        * You can reuse an existing Azure role as long as it has the required minimal permissions necessary for Cloudbreak to be able the manage the cluster's resources. It returns an error if no role with the specified name exists or if the role does not have the required permissions. 
     * `Let Cloudbreak create a custom role`
-        * Choosing this option will let Cloudbreak application create the Azure role with the necessary permissions. It returns an error if a role alerady exists with the name specified.
+        * Choosing this option will let the Cloudbreak application create an Azure role with the necessary permissions. It returns an error if a role with the specified name alerady exists.
```
Comment (pdarvasi) [inline docs/azure/ui/credential.md]: an -> the, I meant that exact role

File: docs/azure/ui/credential.md [hunk 3]
```diff
@@ -1,93 +1,87 @@
...
 
-> The necessary Action set for Cloudbreak to be able to manage the clusters include:
+> The necessary action set for Cloudbreak to be able to manage the clusters include:
```
Comment (pdarvasi) [inline docs/azure/ui/credential.md]: action -> Action, this is Azure terminology

File: docs/azure/ui/credential.md [hunk 4]
```diff
@@ -1,93 +1,87 @@
...
-<sub>*Full size [here](/azure/images/azure-application.png).*</sub>
+    ![](/azure/images/azure-application.png)
+    <sub>*Full size [here](/azure/images/azure-application.png).*</sub>
 
-  - Copy your SSH public key to the `SSH public key` field
-    * The SSH public key must be in OpenSSH format and it's private keypair can be used later to [SSH onto every 
-    instance](operations.md#ssh-to-the-hosts) of every cluster you'll create with this credential.
-    - The **SSH username** for the AZURE instances is **cloudbreak**.
+4. Copy your SSH public key to the `SSH public key` field
+    * The SSH public key must be in the OpenSSH format. The matching private key can be used later to [SSH onto every instance](operations.md#ssh-to-the-hosts) of every cluster that you'll create with this credential. The **SSH username** for the AZzure instances is always **cloudbreak**.
```
Comment (pdarvasi) [inline docs/azure/ui/credential.md]: AZzure typo

**Comment by dbialek2** [comment]
Thanks @pdarvasi! I added your corrections.

**Comment by mhmxs** [comment]
This PR is conflicting. Could you please resolve conflicts or delete it?

**Pull request closed without merging**